pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -615,6 +614,51 @@ were also defined on the `task` level.
 	          - echo $VAR_1
 	          - echo $APP_ENV
 
+### matrix
+
+The `martix` property allows you to define one or more environment variables
+with multiple values. After that, it creates multiple parallel jobs by
+calculating the Cartesian product of all sets of values.
+
+So, the final outcome of the `matrix` property is parallel jobs with the
+commands that are defined in the respective `jobs` property joined with all
+possible combinations of the environment variables defined in the `matrix`
+property.
+
+#### Example of matrix
+
+The following pipeline illustrates the use of the `matrix` property:
+
+	version: v1.0
+	name: Using the matrix property
+	agent:
+	  machine:
+	    type: e1-standard-2
+	    os_image: ubuntu1804
+    
+	blocks:
+	  - name: Elixir + Erlang
+	    task:
+	        jobs:
+	        - name: Elixir + Erlang matrix
+	          commands:
+	            - echo $ELIXIR
+	            - echo $ERLANG
+	          matrix:
+	            - env_var: ELIXIR
+	              values: ["1.3", "1.4"]
+	            - env_var: ERLANG
+	              values: ["19", "20", "21"]
+
+In this example the values of the `matrix` property expand to 6 parallel jobs
+as there are 2 * 3 = 6 combinations of the provided environment variables:
```

Comment:
In this example, job specification named `Elixir + Erlang matrix` expands to 6 parallel jobs
as there are 2 * 3 = 6 combinations of the provided environment variables:

Reply:
Done.